client/ayon_core/tools/utils/widgets.py

```diff
@@ -131,6 +136,29 @@ def __init__(self, *args, **kwargs):
             viewport.setPalette(filter_palette)
 
 
+class MarkdownLabel(QtWidgets.QLabel):
```

Comment:
Is there a reason why [this](https://doc.qt.io/qt-6/qt.html#TextFormat-enum) doesn't suffice?
```python
label = QtWidgets.QLabel("Hello **Markdown**)
label.setTextFormat(QtCore.Qt.MarkdownText)
```

Reply:
From documentation `This enum value was added in Qt 5.14.`, not sure if we match that needs in all DCCs?

Reply:
The first Python 3 releases that I could find for Maya, Nuke and Houdini:
[Maya 2022: 5.15.2](https://help.autodesk.com/view/MAYAUL/2022/ENU/?guid=Maya_SDK_Open_Source_Components_2022_Open_Source_Components_html)
[Nuke 13: QT Python (Pyside) → 5.12.6 (with Patch)](https://learn.foundry.com/nuke/content/release_notes/nuke_13.0.html)
[Houdini 18.5: 5.12.6](https://www.sidefx.com/docs/houdini18.5/licenses/index.html)